client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -0,0 +1,92 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isFreeJetpackPlan } from 'lib/products-values';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class DisconnectSurvey extends Component {
+	state = {
+		reasonSelected: 'tooExpensive',
+		renderInitial: true,
+	};
+
+	renderFollowUp() {
+		// placeholder
+		return <Card className="disconnect-site__question">{ 'follow-up' }</Card>;
+	}
+
+	handleAnswerClick = option => {
+		this.setState( {
+			reasonSelected: option.value,
```

Comment:
This won't work :slightly_frowning_face: We aren't given an `option` here but an `event`. What we can do is provide, say, `data-reason` to `CompactCard`, and access it here via `event.currentTarget.dataset.reason`.